srcpkgs/git-cliff/template

```diff
@@ -11,8 +12,15 @@ license="GPL-3.0-only"
 homepage="https://github.com/orhun/git-cliff"
 changelog="https://raw.githubusercontent.com/orhun/git-cliff/main/CHANGELOG.md"
 distfiles="https://github.com/orhun/git-cliff/archive/refs/tags/v${version}.tar.gz"
-checksum=451db5e7ce904f33cf5772cd67400015829e5b7b3d18955bc3e8f88977b63793
+checksum=045fb10ad0ee340172397f2bb6521c7ee0a83dca2c9f9d177300601f6c60184f
 
 post_install() {
 	vdoc README.md
+
+	mkdir completions
+	OUT_DIR=completions vtargetrun ${DESTDIR}/usr/bin/git-cliff-completions
```

Comment:
```suggestion
	mkdir -p completions
	OUT_DIR=completions vtargetrun ${DESTDIR}/usr/bin/git-cliff-completions
```
But it might be better to leave out the completions directory and put it directly to $wrksrc if that's possible.

Reply:
like so?

```patch
diff --git a/srcpkgs/git-cliff/template b/srcpkgs/git-cliff/template
index 64c55a3c6d..96743e5781 100644
--- a/srcpkgs/git-cliff/template
+++ b/srcpkgs/git-cliff/template
@@ -17,8 +17,7 @@ checksum=045fb10ad0ee340172397f2bb6521c7ee0a83dca2c9f9d177300601f6c60184f
 post_install() {
 	vdoc README.md
 
-	mkdir completions
-	OUT_DIR=completions vtargetrun ${DESTDIR}/usr/bin/git-cliff-completions
+	OUT_DIR=${PWD} vtargetrun ${DESTDIR}/usr/bin/git-cliff-completions
 
 	vcompletion "completions/_git-cliff" zsh
 	vcompletion "completions/git-cliff.fish" fish
```

Reply:
Committed this now